UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.UnitTests/Controllers/EssWebhookControllerTests.cs

```diff
@@ -212,6 +246,40 @@ private EnterpriseEventCacheDataRequest GetCacheRequestData()
             };
         }
 
+        private EnterpriseEventCacheDataRequest GetCacheRequestS57Data()
```

Comment:
Accept BuisnessUnit all details are same in GetCacheRequestS63Data() method can we change GetCacheRequestS63Data() to GetCacheRequestData(string exchangeSetStandard) and set BusinessUnit = exchangeSetStandard to avoid duplicate code and remove GetCacheRequestS57Data() method please.